doc/classes/Transform3D.xml

```diff
@@ -159,111 +164,118 @@
 			<return type="Transform3D" />
 			<param index="0" name="offset" type="Vector3" />
 			<description>
-				Returns a copy of the transform translated by the given [param offset].
+				Returns a copy of this transform translated by the given [param offset].
 				This method is an optimized version of multiplying the given transform [code]X[/code] with a corresponding translation transform [code]T[/code] from the right, i.e., [code]X * T[/code].
 				This can be seen as transforming with respect to the local frame.
 			</description>
```

Comment:
ALL of these descriptions, yes, all of them need a massive change:
`translated`, `rotated`, `scaled` and their equivalent `*_local` methods.

But I have no idea what of this description is meaningful, what NEEDS to be said, and how to phrase it. I need way further feedback.